src/urls.json

```diff
@@ -1,4 +1,12 @@
 { 
-    "get": ["https://duckduckgo.com/", "https://google.com/"], 
-    "ping": ["https://duckduckgo.com/", "https://google.com/"]
+    "get": [
+        "https://api-opis.unictdev.org/", 
+        "https://telegram.unictdev.org/",
+        "https://open-job-dmi.unictdev.org/"
+    ], 
+    "ping": [
+        "https://api-opis.unictdev.org/", 
+        "https://telegram.unictdev.org/",
+        "https://open-job-dmi.unictdev.org/"
+    ]
```

Comment:
mhm, ideally we should move this in the settings yaml conf file, to keep the software "generic and configurable", but we could do this in another PR